File: java/PML/src/tags/TagHTML.java
package tags;

import java.util.ArrayList;

import parser.PMLParser;

public class TagHTML extends Tag{

	@Override
	public void output(ArrayList<Tag> tags) {
		System.out.println("<html>");
		PMLParser.parse(tags);
		System.out.println("</html>");
	}
}

File: php/classes/PML/Tags/TagHTML.php
<?php namespace PML\Tags;

class TagHTML extends Tag {

    /**
     * Construct the base type, Tag
     *
     * @param Array $attributes
     * @param Array $childTags
     */
    public function __construct(Array $attributes, Array $childTags)
    {
        parent::__constrct($attributes, $childTags);
    }

    /**
     * Define some output for this tag
     *
     * @param  Array  $tags
     */
    public abstract function output(Array $tags)
    {
        echo '<html>';
        PMLParser::parse($tags);
        echo '</html>';
    }
}
